syncd/scripts/syncd_init_common.sh

```diff
@@ -26,8 +26,10 @@ else
     CMD_ARGS=
 fi
 
-# Use temporary view between init and apply
-CMD_ARGS+=" -u"
+# Use temporary view between init and apply except of fast-reboot
+if [[ "$(cat /proc/cmdline)" != *"SONIC_BOOT_TYPE=fast-reboot"* ]]; then
```

Comment:
The /proc/cmdline is persistent. Consider I did fast-reboot and then config reload, the /proc/cmdline still has SONIC_BOOT_TYPE=fast-reboot, but syncd is botting in cold mode.

Reply:
Why do you need to disable temporary view here for fast-reboot exclusively? How it is different from fastfast boot, which does not use temporary view?